Updated music.yaml files (RA/TD/D2k)

jaZzKCS opened the pull request:
_Update: All three mod files have now been updated, made sure the correct names appear, made sure that regardless of what version you use (vanilla/expansion/content mirror) you have all files correctly listed, alphabetized and put into corresponding expansion section of the files, if applicable._

Added a one-liner to make sure the song "Afterlife" (await_r.aud) is loaded in both content data alternatives (Aftermath scores.mix from DL mirror / Base game scores.mix file from Game CD.)

Depending of which scores.mix file is used (the DL mirror one including the Aftermath expansion tracks or the vanilla one from "Install from CD"-option)
the naming of the file is slightly different. This results in missing out this song when using the bigger scores.mix file with the bonus tracks, since OpenRA can't find the file "await_r.aud", because in the expansion mix files, it is actually called "await.aud".

Closes #10160.

jaZzKCS committed:
Update music.yaml

Added a one-liner to make sure the song "Afterlife" (await_r.aud) is loaded in both content data alternatives (Aftermath scores.mix from DL mirror / Base game scores.mix file from Game CD.)

Regarding of which scores.mix file is used (the DL mirror one including the Aftermath expansion tracks or the vanilla one from "Install from CD"-option)
the naming of the file is different. Resulting in missing out this song when using the bigger scores.mix file with the bonus tracks.

jaZzKCS commented:
relating to: https://github.com/OpenRA/OpenRA/issues/10160

Mailaender commented on mods/ra/audio/music.yaml:
```diff
@@ -4,6 +4,7 @@ map: Map
 	Hidden: true
 score: Militant Force
 	Hidden: true
+await: Await
```
I suggest you move it downward in the `# Aftermath tracks` section so that information isn't lost.

jaZzKCS commented:
Yes, correct. I have moved it to the Aftermath section.
I have also renamed the song desc. to "Afterlife", since the Red Alert PC gamedata itself suggests that this is the real title according to the "aud tag".

jaZzKCS commented:
Changes made to both music.yaml files (TD/RA)

TD music.yaml categorized in the same manner as the RA music.yaml (separate section for Covert Ops music - now you are able to see which ones are expansion songs).

Alphabetized + corrected some songnames to reflect their in-game/game-data description in both files.
These files are now compatible with both - the vanilla scores.mix from the "Install from CD"-function as well as the scores.mix from Covert Ops/Aftermath (used in content mirrors/Download-function.)

jaZzKCS commented:
Actually, both yamls are now compatible with all _3_ scenarios.
For both CnC games (RA/TD): In all three scenarios (vanilla scores.mix / expansion scores.mix / content download scores.mix) the songlist/filesnames oh so slightly differ.

It is now made sure that regardless of which version you use, all songs you have in your mix file are appropriately listed inside the Music menu.

pchote commented:
I wonder whether this would be a good time to remove the Ride of the Valkyries track from TD.  Player's reactions when they first encounter it is almost universally "WTF?!? was this really in the original game?" (answer: not in a way you could access normally).

jaZzKCS commented:
MAybe hide it from the general playlist so it isnt played in skirmish/MP/missions, yest it is still possible to make us of it in custom missions?

pchote commented:
Yeah, that'll work (set `Hidden: true`, like the end-game music).

LipkeGu commented:
#9162

abcdefg30 commented:
@jaZzKCS could you squash the commits, please?

jaZzKCS commented:
Squashed.

Mailaender commented:
Seems fine in-game. :white_check_mark:

obrakmann commented:
Looks good to me :+1:

obrakmann commented:
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-%28bleed%29/_compare/09bda78376fa481a35d222d60eed52660cd5f93f)
